src/api/user/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "user-microservice",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node server.js",
+    "dev": "nodemon server.js",
+    "emu": "firebase emulators:start --project telescope --only firestore",
+    "jest": "jest",
+    "test": "run-p emu jest"
+  },
+  "jest": {
+    "testEnvironment": "node",
+    "coveragePathIgnorePatterns": [
+      "/node_modules/"
+    ]
+  },
+  "dependencies": {
+    "@senecacdot/satellite": "^1.7.0",
```

Comment:
Satellite is at version 1.8, I think you might wanna update that dependency

Reply:
Use `^1.x`